common/src/main/java/com/wynntils/utils/mc/LoreUtils.java

```diff
@@ -242,6 +242,25 @@ public static boolean isLoreEquals(ListTag existingLore, ListTag newLore) {
         return existingLoreString.equals(newLoreString);
     }
 
+    /**
+     * This checks if the lore of the second item contains the entirety of the first item's lore.
```

Comment:
I'm thinking this should be symmetric, so it also returns true if the first item contains all of the second items lines.

Then it will catch the case of disabling a weapon skin as well.

Or maybe I'm missing something in my thinking here.

Reply:
It shouldn't be possible for a skin getting disabled to cause any conflicts, since it can't coincide with a spell cast - but I might as well make this method symmetric anyway

Reply:
It will save us a re-parse in GearAnnotator when you switch weapon. So yeah, it's not a bug fix, but a (ridiculous) micro-optimization.